lib/motion/configuration.rb

```diff
@@ -59,16 +59,28 @@ def initialize
       Rails.application.key_generator.generate_key("motion:secret")
     end
 
+    option :revision_paths do
+      require "rails"
+
+      paths
+    end
+
     option :revision do
-      warn <<~MSG # TODO: Better message (Focus on "How do I fix this?")
-        Motion is automatically inferring the application's revision from git.
-        Depending on your deployment, this may not work for you in production.
-        If it does, add "config.revision = `git rev-parse HEAD`.chomp" to your
-        Motion initializer. If it does not, do something else (probably read an
-        env var or something).
+      warn <<~MSG
+        Motion is automatically inferring the application's revision by
+        calculating the content of files in motion_config.revision_paths.
+        The defaults revision paths are: Rails paths, bin, and the Gemfile.lock.
+
+        You can add paths to the revision_path using the `revision_paths` option.
+
+        If you prefer to use git for the revision in production,
+        add "config.revision = `git rev-parse HEAD`.chomp" to your
+        Motion initializer.
+
+        Other options might include an environmental variable.
```

Comment:
Thank you for taking the time to rewrite this, but I don't think we need to show this warning anymore if we are computing the hash of the files ourselves (we know that we will be able to do that in production).

Can this copy be incorporated into [the default initializer](https://github.com/unabridged/motion/blob/master/lib/generators/motion/templates/motion.rb)?